lib/data/data_sources/remote_data_source/models/sendbird_base_message.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:sendbird_sdk/core/message/base_message.dart';
+import 'package:sendbird_sdk/core/models/sender.dart';
+import 'package:sendbirdtutorial/domain/entities/chat_message.dart';
+import 'package:sendbirdtutorial/domain/entities/chat_user.dart';
+
+import '../../../../main.dart';
+
+extension SendbirdBaseMessageExtension on BaseMessage {
+  ChatMessage toDomain() => ChatMessage(
+        createdAt: this.createdAt,
+        data: this.data,
+        messageId: this.messageId,
+        message: this.message,
+        channelUrl: this.channelUrl,
+        sender: this.sender != null ? this.sender.senderToUser() : ChatUser(userId: ''),
```

Comment:
which is the meaning of sender = null? if null is possible, I would use null but not creating a User with an empty userId because it's gonna be a wrong user

Reply:
when admin send a message from web dashboard, the sender is null